packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/save-modal.tsx

```diff
@@ -0,0 +1,284 @@
+                                    className='radio-group__save-type'
+                                    name='is_local'
+                                    selected={(() => {
+                                        if (is_authorised) {
+                                            return is_local ? save_types.LOCAL : save_types.GOOGLE_DRIVE;
+                                        }
+
+                                        return save_types.LOCAL;
+                                    })()}
+                                    onToggle={() => setFieldValue('is_local', !is_local)}
+                                >
+                                    <RadioGroup.Item
+                                        id='local'
+                                        label={
+                                            <IconRadio
+                                                text={localize('Local')}
+                                                icon={<Icon icon={is_mobile ? 'IcMobile' : 'IcDesktop'} size={48} />}
+                                            />
+                                        }
+                                        value={save_types.LOCAL}
+                                    />
+                                    <RadioGroup.Item
+                                        id='drive'
+                                        label={
+                                            <IconRadio
+                                                text={'Google Drive'}
+                                                icon={<Icon icon={'IcGoogleDrive'} size={48} />}
+                                                google_drive_connected={is_authorised}
+                                                onDriveConnect={onDriveConnect}
+                                            />
+                                        }
+                                        value={save_types.GOOGLE_DRIVE}
+                                        disabled={!is_authorised}
+                                        className={classNames({
+                                            'dc-radio-group__item-disabled': !is_authorised,
+                                        })}
+                                    />
+                                </RadioGroup>
+                            </div>
+                            <>
+                                <Field name='save_as_collection'>
+                                    {({ field }) => (
+                                        <Checkbox
+                                            {...field}
+                                            onChange={() => setFieldValue('save_as_collection', !save_as_collection)}
+                                            defaultChecked={save_as_collection}
+                                            label={
+                                                <Text color='general' size='xs' line_height='s' weight='bold'>
```

Comment:
general is default color value we can remove

Reply:
done ✅